File: e2e-tests/spring-rest-openapi-v2/src/main/java/com/foo/rest/examples/spring/db/directint/DbDirectIntRepository.java
package com.foo.rest.examples.spring.db.directint;

import org.springframework.data.repository.CrudRepository;

import java.util.List;

public interface DbDirectIntRepository extends CrudRepository<DbDirectIntEntity, Long> {

    List<DbDirectIntEntity> findByXIsAndYIs(Integer x, Integer y);
}

File: e2e-tests/spring-graphql/src/main/kotlin/com/foo/graphql/db/directint/DbDirectIntRepository.kt
package com.foo.graphql.db.directint

import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface DbDirectIntRepository : CrudRepository<DbDirectInt, Long>{
    fun findByXAndY(x : Int, y : Int) : List<DbDirectInt>
}